src/backend/distributed/citus--7.0-8--7.0-9.sql

```diff
@@ -0,0 +1,12 @@
+/* citus--7.0-8--7.0-9 */
+
+SET search_path = 'pg_catalog';
+
+CREATE FUNCTION check_distributed_deadlocks()
```

Comment:
Hmm, initially it helped me for testing & debugging purposes. But, you're right, we don't need to expose this I think.